cvat/apps/events/handlers.py

```diff
@@ -84,26 +84,43 @@ def _get_current_user(instance=None):
 
     return None
 
-def user_id(instance=None):
-    current_user = _get_current_user(instance)
-    if current_user is not None:
-        return getattr(current_user, "id", None)
+def _get_current_request(instance=None):
+    request = get_current_request()
+    if request is not None:
+        return request
+
+    if isinstance(instance, rq.job.Job):
+        return instance.meta.get("request", None)
+    else:
+        rq_job = rq.get_current_job()
+        if rq_job:
+            return rq_job.meta.get("request", None)
```

Comment:
Should we first check if the instance is rq job?
If I try to stage a failed file import then I will see in Grafana Dashboard
```
{"message":"Exception: ..." ...,"request":{"id":"33ea5cc9-1dc7-48c4-b29f-6c1f3fd4daeb"}}
```
request with id `33ea5cc9-1dc7-48c4-b29f-6c1f3fd4daeb` but likely to be expected `615b99d2-8435-40b8-878d-6314ed6204ec` (id of failed request)
![image](https://user-images.githubusercontent.com/49038720/224764770-6d9427cd-5df1-4ae6-b08a-33846b5c9664.png)
![image](https://user-images.githubusercontent.com/49038720/224765008-c8fef515-6606-40af-9a1f-f4a6cf97e250.png)